packages/auth/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.compilerOption.json",
+  "compilerOptions": {
+    "tsBuildInfoFile": "dist/tsconfig.tsbuildinfo",
+    "baseUrl": ".",
+    "outDir": "dist",
+    "rootDir": "src",
+    // gotrue-js does not return types for `removeSavedSession` and `recoverSession`
+    "noImplicitAny": false,
```

Comment:
Boo. Could we extend with a `.d.ts` module? Alternatively I'd be happy to contribute by adding those types upstream to gotrue

Reply:
Ah, this is my first _real_ TS project and I had some trouble figuring out what to do in this case - it seemed like contributing upstream was the recommended way, but hadn't considered fixing it via a `.d.ts` file.
 
I'll give that a go!